pyravendb/tools/utils.py

```diff
@@ -291,7 +292,11 @@ def timedelta_to_str(timedelta_obj):
                 timedelta_str += "{0}.".format(days)
             timedelta_str += "{:02}:{:02}:{:02}".format(hours, minutes, seconds)
             if microseconds > 0:
-                timedelta_str += ".{0}".format(microseconds)
+                timedelta_str += "."
+                order_of_magnitude = log10(microseconds).__trunc__()
```

Comment:
wouldn't `math.trunc()` be more "elegant" instead?

Reply:
done